README.md

```diff
@@ -354,8 +348,7 @@ You can change the default folder by setting the `NOSTR_CONFIG_DIR` environment
 Run nostream using one of the quick-start guides at least once and `~/.nostr/settings.json` will be created.
 Any changes made to the settings file will be read on the next start.
 
-A sample settings file is included at the project root under the name `settings.sample.json`. Feel free to copy it to `~/.nostr/settings.json`
-if you would like to have a settings file before running the relay first.
+Default settings can be found under `resources/default-settings.yaml`. Feel free to copy it to `~/.nostr/settings.yaml` if you would like to have a settings file before running the relay first.
```

Comment:
Should this say:
`Feel free to copy it to `nostream/.nostr/settings.yaml` if you would like to have a settings file before running the relay first.

Reply:
@Cameri just double checking this is correct.